### PR TITLE
feat: not realize building

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

**Review by pgallik — APPROVED** [review]

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/NotRealizeBuildingRequestValidator.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.Building.Validators
+{
+    using FluentValidation;
+    using Requests;
+
+    public class NotRealizeBuildingRequestValidator: AbstractValidator<NotRealizeBuildingRequest>
+    {}
```
Comment (lvermeulen) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/NotRealizeBuildingRequestValidator.cs:7]: space between braces

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs [hunk 2]
```diff
@@ -33,6 +34,7 @@ public static class Building
             public const string InvalidPolygonGeometry = "Ongeldig formaat geometriePolygoon.";
             public const string BuildingCannotBePlacedUnderConstruction = "Deze actie is enkel toegestaan op gebouwen met status 'gepland'.";
             public const string BuildingCannotBeRealizedException = "Deze actie is enkel toegestaan op gebouwen met status 'inAanbouw'.";
+            public const string BuildingCannotBeNotRealizedException = "Deze actie is enkel toegestaan op gebouw met status 'inAanbouw','gepland'.";
```
Comment (lvermeulen) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs:37]: space between 'inAanbouw','gepland"

File: src/BuildingRegistry/Building/Exceptions/BuildingStatusPreventsNotRealizeBuildingException.cs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+namespace BuildingRegistry.Building.Exceptions
+{
+    public class BuildingStatusPreventsNotRealizeBuildingException : BuildingRegistryException
+    {}
```
Comment (lvermeulen) [inline src/BuildingRegistry/Building/Exceptions/BuildingStatusPreventsNotRealizeBuildingException.cs:4]: space between braces

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/NotRealizeBuildingRequestValidator.cs (hunk 1 above)
Comment (emalfroy) [reply src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/NotRealizeBuildingRequestValidator.cs]: done

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs (hunk 2 above)
Comment (emalfroy) [reply src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs]: done

File: src/BuildingRegistry/Building/Exceptions/BuildingStatusPreventsNotRealizeBuildingException.cs (hunk 3 above)
Comment (emalfroy) [reply src/BuildingRegistry/Building/Exceptions/BuildingStatusPreventsNotRealizeBuildingException.cs]: done

**Pull request merged**